### PR TITLE
About page style & minor wording change.

### PR DESCRIPTION
**Pull request by MC42**
* Added extra sentence about new stuff TBA offers (insights + blog + CAD stuff.)
* Changed out links to be buttons.

image

### CODE REVIEW

**Comment by nwalters512** [comment]
Looks good! Maybe put more space between the icons and text in the buttons?

**Comment by MC42** [comment]
image

Added some extra spaces in to distance it out, I think it works.

**Comment by fangeugene** [comment]
LGTM

File: templates/about.html [hunk 1]
```diff
@@ -15,9 +15,18 @@ <h1 class="end_header">About</h1>
...
-      resource for our users.</p>
-      <p>Have a suggestion? <a href="/contact">Contact us!</a></p>
-      <p>Are you a developer? Check out our <a href="https://github.com/the-blue-alliance/the-blue-alliance" target="_blank"> GitHub repository</a> and get involved!</p>
+      resource for our users.  Other additions to The Blue Alliance include match prediction and anaytics, 
+      hosting of CAD models, and The Blue Alliance Blog.</p>
+	   
+      <div class="hidden-xs">
+      <br>
+      <div class="btn-group btn-group-block-2">
+<a class="btn btn-default" href="/contact"><span class="glyphicon glyphicon-info-sign"></span>    Contact Us!</a>
```
Comment (fangeugene) [inline templates/about.html]: I don't believe there's any difference between 1 space and multiple spaces. Same below.

**Pull request merged**